resolwe/flow/managers/listener.py

```diff
@@ -373,7 +373,8 @@ def handle_referenced_files(self, obj):
                 if data.status == Data.STATUS_PROCESSING:
                     data.status = Data.STATUS_DONE
                     data.save()
-                    validate_data_object(data)
+                    # Do not raise error if input data object is missing.
```

Comment:
Mention that we can skip missing data as the lock guarantee that the storage location is still there.

Reply:
Done.